lib/OrdinaryDiffEqRosenbrock/src/stiff_addsteps.jl

```diff
@@ -558,54 +558,22 @@ function _ode_addsteps!(k, t, uprev, u, dt, f, p, cache::Rosenbrock5ConstantCach
     nothing
 end
 
-function _ode_addsteps!(k, t, uprev, u, dt, f, p, cache::Rosenbrock5Cache,
```

Comment:
This function also should be deleted in it's entirety (for the same reason as perform step).